src/python/WMCore/REST/Format.py

```diff
@@ -46,9 +49,14 @@ def __call__(self, stream, etag):
         # to a single string. Convert files to 1MB chunks.
         if stream is None:
             stream = ['']
-        elif isinstance(stream, basestring):
+        elif isinstance(stream, str) or isinstance(stream, bytes):
```

Comment:
This comparison can be made against a tuple, something like
```suggestion
        elif isinstance(stream, (str, bytes)):
```

Reply:
You are correct! I pushed this change in a new commit